client/src/components/admin/AdminCompetitionEvents/ClearRoundButton.jsx

```diff
@@ -9,6 +9,8 @@ const CLEAR_ROUND_MUTATION = gql`
       round {
         id
         open
+        enteredResults
+        totalResults
```

Comment:
I added this for both open and clear. The reason for including particular fields in the graphql response is that the local cache and therefore existing queries are immediately updated. Without this, after opening a round we would see `0 of 0 entered` :)